build_targets

```diff
@@ -14,6 +15,8 @@ m52xq user yes yes
 mars userdebug yes yes
 mayfly userdebug yes yes
 miatoll userdebug yes yes
+nx_tab userdebug yes yes
+oriole userdebug yes yes
```

Comment:
check dependencies
tracking invalid repo

Reply:
which repo is invalid? at least for me all of them are valid

Reply:
https://github.com/crdroidandroid/android_device_google_oriole/blob/14.0/crdroid.dependencies#L12

Reply:
> https://github.com/crdroidandroid/android_device_google_oriole/blob/14.0/crdroid.dependencies#L12

im able to access it fine -- even when signed out
![image](https://github.com/user-attachments/assets/4388159a-652c-4083-8a59-99f6baf68543)

Reply:
it's not allowed 
see readme

Reply:
> it's not allowed see readme

So sorry, I thought you meant you couldn't access the repo. I have spare time now so I'll begin work on cloning all required repositories for oriole, guacamoleb and nx_tab to the crdroidandroid github.

Reply:
So, severe issue, every time I try to upload the raviole-kernel to the crdroidandroid github, I get a 500 error. I'm assuming this is due to the repository being like 44gb with it's full commit history. This is actually the reason it's on my personal GitHub, I kinda expected this and I didn't want to publish a repository with no commits to the crdroidandroid github.

Reply:
> So, severe issue, every time I try to upload the raviole-kernel to the crdroidandroid github, I get a 500 error. I'm assuming this is due to the repository being like 44gb with it's full commit history. This is actually the reason it's on my personal GitHub, I kinda expected this and I didn't want to publish a repository with no commits to the crdroidandroid github.

What am I supposed to do in this case? Should I upload it with no history or should I just remove oriole for now until I can get this figured out?

Reply:
keep only latest few commits history